debian/appscale_install_functions.sh

```diff
@@ -160,6 +160,17 @@ root            soft    nofile           200000
 *               soft    nofile           200000
 *               -       nproc            32768
 EOF
+
+    # On distros with systemd, the open file limit must be adjusted for each
+    # service.
+    if which systemctl; then
+        mkdir -p /etc/systemd/system/nginx.service.d
+        cat <<EOF | tee /etc/systemd/system/nginx.service.d/override.conf
```

Comment:
Why the 'tee' here? Do we need to see the output of this command?

Reply:
Good point. I don't think the output is helpful.